test/uk/gov/hmrc/cataloguefrontend/healthindicators/HealthIndicatorsControllerSpec.scala

```diff
@@ -333,8 +333,8 @@ class HealthIndicatorsControllerSpec
         | "weightedMetrics": []
         |}]""".stripMargin
 
-    val createdAt    = LocalDateTime.of(2016, 5, 23, 16, 45, 30)
-    val lastActiveAt = LocalDateTime.of(2016, 10, 12, 10, 30, 12)
+    val createdAt    = LocalDateTime.of(2016, 5, 23, 16, 45, 30).toInstant(ZoneOffset.UTC)
```

Comment:
Can we replace these with the `Instant.parse("2016-05-23T16:45:30.00Z")` format?